Automattic/wp-calypso
# Gutenberg: Fix "Block settings" styling issues

#### Changes proposed in this Pull Request

Fixes #28148.

In #28095 we started to import the styles from `@wordpress/edit-post` rather than from the local copy we had. 

We updated the version of the package in #28000, but we didn't update the copied JS files we still have, so it was causing some styling issues on the Block settings in the sidebar because the imported styles are intended to be used with a different version of the JS files.

The styles expect to render a [`SettingsSidebar` component](https://github.com/WordPress/gutenberg/blob/a2f81faa58afbbcb28d68ef879a5354c257b2d85/packages/edit-post/src/components/layout/index.js#L116), but our copied version of the `edit-post` package is rendering [`DocumentSidebar` and `BlockSidebar` instead](https://github.com/Automattic/wp-calypso/blob/d22161008de4143a4b4d161a177ab17b71c99ac6/client/gutenberg/editor/edit-post/components/layout/index.js#L101-L102).

This PR updates the needed files, so the styles and the JS file are sync and matches the behavior of the current published package.

#### Testing instructions
1. Start writing a post using the Gutenberg editor
2. Add a paragraph block
3. Check the Block settings on the sidebar and make sure it looks like the After screenshot below.

#### Screenshots
Before
<img width="280" alt="screen shot 2018-10-29 at 15 26 23" src="https://user-images.githubusercontent.com/1233880/47657801-0cb9a000-db92-11e8-9693-de165ad66b8a.png">

After
<img width="281" alt="screen shot 2018-10-29 at 15 47 36" src="https://user-images.githubusercontent.com/1233880/47657807-10e5bd80-db92-11e8-857c-74ced77557a1.png">
